src/src/facilitator/facilitator.service.ts

```diff
@@ -81,6 +81,72 @@ export class FacilitatorService {
 		const limit = isNaN(body.limit) ? 15 : parseInt(body.limit);
 
 		let skip = page > 1 ? limit * (page - 1) : 0;
+		let filterQueryArray = [];
+
+		let status;
+		switch (body?.type) {
+			case 'pragati_orientation': {
+				status = `status: { _in: ["applied" ]}`;
+				break;
+			}
+			case 'gkp_training': {
+				status = `status: { _in: ["pragati_mobilizer" ]}`;
+				break;
+			}
+			case 'pcr_training': {
+				status = `status: { _in: ["pragati_mobilizer","selected_for_onboarding" ]}`;
+				break;
+			}
+			case 'main_camp_execution_training': {
+				status = `status: { _in: ["selected_for_onboarding" ]}`;
+				break;
+			}
+			case 'drip_training': {
+				status = `status: { _in: ["applied","pragati_mobilizer","selected_for_onboarding" ]}`;
+				break;
+			}
+		}
+
+		filterQueryArray.push(
+			`	program_faciltators: {
+				parent_ip: { _eq: "${user?.data?.program_users[0]?.organisation_id}" },academic_year_id:{_eq:${academic_year_id}},program_id:{_eq:${program_id}},
```

Comment:
user?.data?.program_users[0]?.organisation_id} need to add filter `program_id and academic_year_id` in program_users

Reply:
when we apply academic and program_id it gives undefined